go/arrow/compute/arithmetic_test.go

```diff
@@ -2621,7 +2621,7 @@ func (us *UnaryArithmeticSigned[T]) TestAbsoluteValue() {
 				fn(`[]`, `[]`)
 				// scalar/arrays with nulls
 				fn(`[null]`, `[null]`)
-				fn(`[1, null -10]`, `[1, null, 10]`)
+				fn(`[1, null, -10]`, `[1, null, 10]`)
```

Comment:
so this is interesting, how was this not failing already?

Reply:
The goccy JSON parser seems to be more forgiving than the built-in one.